parsing/HACKING.adoc

```diff
@@ -25,6 +25,52 @@ target of the root Makefile to rebuild the compiler parser. See
 link:../Makefile.menhir[] for the details of the various
 Menhir-related targets and their use.
 
+In addition, if your purpose is to modify the parser, it is recommended that
+you make sure to use the same version of Menhir to generate the
```

Comment:
"In addition, if your purpose if to modify the parser," is not so useful here, given that this section precisely explains how to modify the parser, so it should be clear from the context.